setup.py

```diff
@@ -1,526 +1,29 @@
-    metadata, console_scripts = read_setup_cfg()
-
-    command_options = {
-        'docs': {'project': ('setup.py', package),
-                 'version': ('setup.py', version.split('-', 1)[0]),
-                 'release': ('setup.py', version),
-                 'build_dir': ('setup.py', docs_build_path),
-                 'config_dir': ('setup.py', docs_path),
-                 'source_dir': ('setup.py', docs_path)},
-        'doctest': {'project': ('setup.py', package),
-                    'version': ('setup.py', version.split('-', 1)[0]),
-                    'release': ('setup.py', version),
-                    'build_dir': ('setup.py', docs_build_path),
-                    'config_dir': ('setup.py', docs_path),
-                    'source_dir': ('setup.py', docs_path),
-                    'builder': ('setup.py', 'doctest')},
-        'test': {'test_suite': ('setup.py', 'tests'),
-                 'cov': ('setup.py', root_pkg)}}
-
-    setup(name=package,
-          version=version,
-          url=metadata['url'],
-          description=metadata['description'],
-          author=metadata['author'],
-          author_email=metadata['author_email'],
-          license=metadata['license'],
-          long_description=read('README.rst'),
-          classifiers=metadata['classifiers'],
-          test_suite='tests',
-          packages=setuptools.find_packages(exclude=['tests', 'tests.*']),
-          namespace_packages=namespace,
-          install_requires=install_reqs,
-          setup_requires=['six'],
-          cmdclass=cmdclass,
-          tests_require=['pytest-cov', 'pytest'],
-          include_package_data=True,
-          package_data={package: metadata['package_data']},
-          data_files=[('.', metadata['data_files'])],
-          command_options=command_options,
-          entry_points={'console_scripts': console_scripts},
-          zip_safe=False)  # do not zip egg file after setup.py install
-
-
-if __name__ == "__main__":
-    setup_package()
+# coding=utf-8
+
+from setuptools import setup, find_packages
+
+with open('README.rst') as f:
+    readme = f.read()
+
+setup(
+    name='cee_syslog_handler',
+    use_scm_version=True,
+    setup_requires=['setuptools_scm'],
+    packages=find_packages(exclude=['tests']),
+    author='Blue Yonder GmbH',
+    author_email='peter.hoffmann@blue-yonder.com',
+    url='https://github.com/blue-yonder/cee_syslog_handler',
+    description='Python Syslog Logging Handler with CEE Support',
+    long_description=readme,
+    license='new BSD',
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Environment :: Console',
+        'Intended Audience :: Developers',
+        'Intended Audience :: System Administrators',
+        'License :: OSI Approved :: BSD License',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3.4',
```

Comment:
You should probably also add the classifier for Python 3.6 as you added that one to the test.

Reply:
Added. Thanks.